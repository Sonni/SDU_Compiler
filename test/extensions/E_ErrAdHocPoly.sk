type t_int = int;

func f(a: t_int): bool
    return true;
end f

func f(a: t_int): int
    return 3;
end f



write f(40);