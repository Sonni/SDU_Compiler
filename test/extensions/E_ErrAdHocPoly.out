Typer checker should output an error